Game/HitShapes/HitShapeSpawnerFunctionality.h

```diff
@@ -14,19 +17,19 @@ enum class EDirection
 class HitShapeSpawnerFunctionality
 	: public Functionality<
 	HitShapeSpawnerFunctionality,
-	Policy::Set<
-	Policy::Add<TransformFunctionality, Policy::EPermission::Write>,
-	Policy::Add<FactionFunctionality, Policy::EPermission::Read>,
-	Policy::Add<SFactionComponent, Policy::EPermission::Write>,
-	Policy::Add<SLevelDenizenComponent, Policy::EPermission::Write>>>
+		Policy::Set<
+			Policy::Add<Randomizer, Policy::EPermission::Write>,
+			Policy::Add<FunctionalitySystemDelegate<HitShapeFunctionality>, Policy::EPermission::Write>,
+			Policy::Add<TransformFunctionality, Policy::EPermission::Write>,
+			Policy::Add<FactionFunctionality, Policy::EPermission::Read>,
+			Policy::Add<SFactionComponent, Policy::EPermission::Write>,
+			Policy::Add<SLevelDenizenComponent, Policy::EPermission::Write>>>
```

Comment:
Feels like we could simplify Policy::Add<TPolicyType, Policy::EPermission::Write> to something like Write<TPolicyType> to reduce boilerplate

Reply:
Agreed, will add them to the global scope and rename them.